Tasks/DownloadSecureFileV1/task.json

```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 126,
+        "Minor": 127,
```

Comment:
We have a weird versioning scheme; the minor version is our current sprint number (it's how we keep track of deployments).  So this should be 137 now.

Reply:
Please also commit the generated task.loc.json file